package.json

```diff
@@ -6,7 +6,7 @@
 	"license": "GPL-2.0-or-later",
 	"private": true,
 	"dependencies": {
-		"@wordpress/scripts": "27.1.0"
+		"@wordpress/scripts": "28.3.0"
```

Comment:
I assume that the code is going to work with WordPress 6.7 because otherwise, it would fail with the missing JSX runtime script dependency.

Reply:
Good callout!

[The v28 changelog reads:](https://github.com/WordPress/gutenberg/blob/eea7183c6b156610cd3daa538e98a8b4b5388c2e/packages/scripts/CHANGELOG.md#2800-2024-05-31)

> **Breaking Changes**
> Note If you're using @wordpress/scripts for building JS scripts to target WordPress 6.5 or earlier, you should not upgrade to this version and continue using @wordpress/scripts@27.

I think the site uses latest WordPress, so if it's on WP 6.6, it should work if that comment is accurate. Do you know if it's WordPress 6.6 or 6.7 that will use the new JSX runtime?

@ryelle can you confirm the site runs on latest WordPress release (6.6.x at this time)?

Reply:
Right, 6.6 is fine, too. I assume that it's sartisfied here. However, I wanted to ensure we don't see any surprises when deploying.

Reply:
WordPress.org runs trunk & (usually) the latest Gutenberg version (the exception is the Support Forums due to conflicts with [blocks-everywhere](https://github.com/Automattic/blocks-everywhere), so we used a pinned version there). So we should be set with this version change. It also worked fine on my sandbox, which matches production versions.